openspec/changes/seo-issues/qa-report.md

```
@@ -0,0 +1,138 @@
+# Acceptance QA Report: seo-issues
+
+## Identity
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)